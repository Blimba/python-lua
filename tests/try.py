try:
    print('test')
    xpcall()
    print('still going')
except:
    print('Error in function')
print('running')