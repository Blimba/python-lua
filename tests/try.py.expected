test
Error in function
running
